src/Native/System.Native/CMakeLists.txt

```diff
@@ -31,8 +31,17 @@ add_library(System.Native
     ${NATIVE_SOURCES}
 )
 
+
+add_library(System.Native.a
+    STATIC
+    ${NATIVE_SOURCES}
+)
+SET_TARGET_PROPERTIES(System.Native.a PROPERTIES PREFIX "")
```

Comment:
It was done in the initial commit of the Unix shims https://github.com/dotnet/corefx/pull/2445 . There was some discussion about it, but I am not able to find it.